src/graph/unit_graph.cc

```diff
@@ -738,7 +755,26 @@ class UnitGraph::CSR : public BaseHeteroGraph {
     return adj_;
   }
 
+  bool Load(dmlc::Stream* fs) {
+    auto meta_imgraph = Serializer::make_shared<ImmutableGraph>();
+    CHECK(fs->Read(&meta_imgraph)) << "Invalid meta graph";
+    meta_graph_ = std::dynamic_pointer_cast<GraphInterface>(meta_imgraph);
+    CHECK(fs->Read(&sorted_)) << "Invalid boolean for sorted_";
+    CHECK(fs->Read(&adj_)) << "Invalid adj matrix";
+    return true;
+  }
+  void Save(dmlc::Stream* fs) const {
+    auto meta_graph_ptr = ImmutableGraph::ToImmutable(meta_graph());
+    fs->Write(meta_graph_ptr);
+    fs->Write(sorted_);
```

Comment:
BTW, I don't think this `sorted_` flag is used anywhere. Would you help double check and delete it?

Reply:
This is added in https://github.com/dmlc/dgl/pull/886.
@zheng-da Is it necessary in this class?

Reply:
Do note that the `sorted_` flag is now directly in the `CSRMatrix` structure, so technically it the one in `CSR` graph is redundant.